ps_contactinfo.tpl

```diff
@@ -52,7 +52,7 @@
       <br>
       {* First tag [1][/1] is for a HTML tag. *}
       {l
-        s='Email: %email%'
+        s='Email: <a href="mailto:%email%">%email%</a>'
```

Comment:
Please make the same modifications as in https://github.com/PrestaShop/PrestaShop/pull/8060